pkg/authorize/roundtripper.go

```diff
@@ -0,0 +1,49 @@
+package authorize
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+type ServerRotatingRoundTripper struct {
+	endpoint     *url.URL
+	initialToken string
+	tokenStore   tokenStore
+
+	wrapper http.RoundTripper
```

Comment:
wrapper communicates the opposite relationship between the ServerRotatingRoundTripper and the underlying RoundTripper. Something like `inner` would be more appropriate. In any case, we should decide on something and use it consistently also with the DebugRoundTripper

Reply:
agreed, I didn't change the naming in the refactoring, but `next` would be more appropriate.